utils/create-component/createComponent.js

```diff
@@ -51,12 +52,12 @@ const questions = [
       'Popups',
     ],
   },
-  {
-    type: 'confirm',
-    name: 'publicModule',
-    message: 'Make access public when publishing?:',
-    default: true,
-  },
+  // {
+  //   type: 'confirm',
+  //   name: 'publicModule',
+  //   message: 'Make access public when publishing?:',
+  //   default: true,
+  // },
```

Comment:
double-checking: Is this intentional?

Reply:
Yes - after slash imports we no longer need this since all the components are bundled together. The public setting for npm will be based on the higher level packages (`react`, `labs-react`, etc.) rather than each individual component. I left it here as a reference, but could remove it if you prefer.

Reply:
Gotcha. Yeah, it's in source control, so we can always go back to find it. As it stands, I don't know why it's commented out.